.github/actions/build_vgg_resnet_action/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Setup and build networks; VGG16, Resnet50
+description: Build and run VV16 and Resnet50 with oneAPI base toolkit and oneAPI-Construction-Kit
+
+inputs:
+  workspace:
+    description: 'Main workflow workspace'
+    default: ${{ github.workspace }}
+
+runs:
+  # We don't want a new docker just a list of steps, so mark as composite
+  using: "composite"
+  steps:
+    - name: setup python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+
+    - name: Install Python libraries and system dependencies
+      shell: bash
+      run: |
+        pip install requests numpy h5py Pillow
+        sudo add-apt-repository universe
+        sudo apt-get update
+        sudo apt-get install -y libblas-dev libopenblas64-dev libopenblas-dev libpthread-stubs0-dev libboost-all-dev
+
+    - name: Clone portDNN
+      shell: bash
+      run: git clone --recursive https://github.com/codeplaysoftware/portDNN.git
```

Comment:
Can we use `--depth 1` and `--single-branch` to reduce how much data we fetch here? Or use `wget` to fetch an archive?

Reply:
Good call. :+1: